js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/ParentUpdatedLink.tsx

```diff
@@ -0,0 +1,150 @@
+import {
```

Comment:
Are these basically the same as the existing dialogs?

Reply:
The new dialog distinguishes 2 lists of asset keys- updated assets and willUpdate asssets

Reply:
@hellendag I don't have a strong opinion on if we should make the modal a generic component, but doing so would take me a while. Could we
- land this so we can surface the details now, and punt on making it generic
- or have you commandeer and have your way with it?